colorcet/plotting.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import holoviews as hv
+from holoviews import opts
+
+from . import get_aliases, all_original_names, palette, cm
+
+array = np.meshgrid(np.linspace(0, 1, 256), np.linspace(0, 1, 10))[0]
+
+def swatch(name, cmap=None, bounds=None, array=array, **kwargs):
+    """Show swatch using matplotlib or bokeh via holoviews"""
+    import holoviews as hv; from holoviews import opts
+
+    title = name if cmap else get_aliases(name)
+    if bounds is None:
+        bounds = (0, 0, 256, 1)
+
+    plot = hv.Image(array, bounds=bounds, group=title)
+    backends = hv.Store.loaded_backends()
+    if 'bokeh' in backends:
+        plot.opts(opts.Image(backend='bokeh', width=900, height=100, toolbar='above',
+                             default_tools=['xwheel_zoom', 'xpan', 'save', 'reset'],
+                             cmap=cmap or palette[name]))
+    if 'matplotlib' in backends:
+        plot.opts(opts.Image(backend='matplotlib', aspect=15, fig_size=350,
+                             cmap=cmap or cm[name]))
+    return plot.opts(opts.Image(xaxis=None, yaxis=None), opts.Image(**kwargs))
+
+def swatches(*args, group=None, not_group=None, only_aliased=False, cols=1, **kwargs):
+    """Show swatches for given names or names in group"""
+    import holoviews as hv; from holoviews import opts
+
+    args = args or all_original_names(group=group, not_group=not_group,
+                                      only_aliased=only_aliased)
+    plot = hv.Layout([
+      swatch(arg, **kwargs) if isinstance(arg, str) else
+      swatch(*arg, **kwargs) for
+      arg in args]).cols(cols)
+
+    backends = hv.Store.loaded_backends()
```

Comment:
@jlstevens , it would be nice if backends not loaded would simply not validate, to avoid having to have code like this that checks the loaded_backends. Maybe something that can go into this hv release?